quickfixj-core/src/main/java/quickfix/mina/LoggingSettings.java

```diff
@@ -0,0 +1,36 @@
+/*******************************************************************************
+ * Copyright (c) quickfixengine.org  All rights reserved.
+ *
+ * This file is part of the QuickFIX FIX Engine
+ *
+ * This file may be distributed under the terms of the quickfixengine.org
+ * license as defined by quickfixengine.org and appearing in the file
+ * LICENSE included in the packaging of this file.
+ *
+ * This file is provided AS IS with NO WARRANTY OF ANY KIND, INCLUDING
+ * THE WARRANTY OF DESIGN, MERCHANTABILITY AND FITNESS FOR A
+ * PARTICULAR PURPOSE.
+ *
+ * See http://www.quickfixengine.org/LICENSE for licensing information.
+ *
+ * Contact ask@quickfixengine.org if any conditions of this licensing
+ * are not clear to you.
+ ******************************************************************************/
+
+package quickfix.mina;
+
+/**
+ * This class holds the QFJ settings information related to logging while session is unavailable.
+ */
+public class LoggingSettings {
+
+    private final boolean logGarbledMessage;
+
+    public LoggingSettings(final boolean logGarbledMessage) {
+        this.logGarbledMessage = logGarbledMessage;
```

Comment:
Just as a side note: a garbled message is something completely different.
https://www.fixtrading.org/standards/fix-session-layer-online/#garbled-message-processing